runtime/src/bank.rs

```diff
@@ -4170,12 +4209,9 @@ impl Bank {
         self.rc.accounts.load_by_program_slot(self.slot(), None)
     }
 
-    pub fn get_account_modified_since_parent(
-        &self,
-        pubkey: &Pubkey,
-    ) -> Option<(AccountSharedData, Slot)> {
+    fn get_account_for_incinerator(&self, pubkey: &Pubkey) -> Option<(AccountSharedData, Slot)> {
```

Comment:
- Removed `pub`
- And renamed as it's only used by `incinerator` to prevent this helper fn from accidentally being used outside on-chain.

Reply:
This seems like a potentially useful utility function, maybe the more generic name `get_account_modified_since_parent(is_fixed_root: bool)` makes sense?